boards/BMS/Test/Src/Test_StateMachine.cpp

```diff
@@ -128,19 +132,19 @@ class BmsStateMachineTest : public BaseStateMachineTest
             DIE_TEMP_TO_DISABLE_CELL_BALANCING_DEGC,
             DIE_TEMP_TO_DISABLE_CHARGER_DEGC);
 
-        air_negative = App_SharedBinaryStatus_Create(is_air_negative_on);
-        air_positive = App_SharedBinaryStatus_Create(is_air_positive_on);
-
         pre_charge_sequence =
             App_PreChargeSequence_Create(enable_pre_charge, disable_pre_charge);
 
+        airs = App_Airs_Create(
+            is_air_positive_closed, is_air_negative_closed, close_air_positive,
+            open_air_positive);
+
         clock = App_SharedClock_Create();
 
         world = App_BmsWorld_Create(
             can_tx_interface, can_rx_interface, imd, heartbeat_monitor,
             rgb_led_sequence, charger, bms_ok, imd_ok, bspd_ok, accumulator,
-            cell_monitors, air_negative, air_positive, pre_charge_sequence,
-            clock);
+            cell_monitors, airs, pre_charge_sequence, clock);
```

Comment:
Haha I wonder if this PR began after accidentally typing air_positive before air_negative in one of the method calls